source/thisTarget.ts

```diff
@@ -41,21 +42,23 @@ import { pEvent } from "p-event";
 // If a message is received before this is ready, it will just have to be ignored.
 export const thisTarget: KnownTarget = isBackground()
   ? { page: "background" }
```

Comment:
Consider an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) vs. nested ternary for readability?